events/sendCustomersRecapEmail.js

```diff
@@ -69,6 +69,7 @@ const sendCustomersRecapEmail = async ({userId}) => {
 							}
 						}, {
 							projects_some: {
+								status: ONGOING
```

Comment:
```suggestion
								status_in: [ONGOING, ARCHIVED]
```

For the recap we can keep archived project I think.